swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -35,56 +35,82 @@ Follow the instructions given in this section to develop the service.
     $ bal new sending-a-message-to-a-service
     ```
 
-2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
+2. Remove the generated content in the `main.bal` file and open the [Ballerina HTTP API Designer](https://wso2.com/ballerina/vscode/docs/design-the-services/http-api-designer) in VS Code.
 
-    Import the 
-    - `ballerina/http` module to develop the REST API and define the client that can be used to send requests to the backend service
-    - `ballerina/log` module to log some information for each client request
-
-    ```ballerina
-    import ballerina/http;
-    import ballerina/log;
-    ```
+    ![Open diagram view](/learn/images/tutorial_sending_a_message_to_a_service_open_diagram_view.gif)
 
 3. Define two [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port on which the listener should listen and the URL of the backend service.
 
+    ![Define configurable variables](/learn/images/tutorial_sending_a_message_to_a_service_define_configurable_variables.gif)
```

Comment:
@sm1990 Can we have a directory structure within `/learn/images`? Then we can avoid this kind of file naming.

Reply:
+1. 

We can have `/learn/images/integration-tutorials/sending_a_message_to_a_service/define_configurable_variables.gif`

Reply:
> @sm1990 Can we have a directory structure within `/learn/images`? Then we can avoid this kind of file naming.

@anupama-pathirage yes.
@dulajdilshan you can create a directory [here](https://github.com/ballerina-platform/ballerina-dev-website/tree/master/public/learn/images)